src/libraries/System.Runtime.Experimental/System.Runtime.Experimental.sln

```diff
@@ -0,0 +1,85 @@
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
Maybe we should just have the System.Runtime.sln  include these as well?  Does it make sense to have a separate sln?

Reply:
It makes it very easy to separate when everything is over, basically just delete one folder rather than try to decouple it from the main solution.

Reply:
Fair enough.

Reply:
You may want to double check that System.Runtime.sln still builds without adding this project to it.